### ISSUE TITLE
Upgrade RHOAI in a test cluster to v2.5

### ISSUE THREAD

**Issue by dystewart** [nerc-project/operations#528]
We currently  only have version 1.33 of rhoai in nerc-ocp-prod and nerc-ocp-test clusters which is blocking our ability to test and utilize mig with rhods workloads. 

This is blocking [https://github.com/nerc-project/operations/issues/501](https://github.com/nerc-project/operations/issues/501)

**Comment by joachimweyl** [comment]
blocked by https://github.com/nerc-project/operations/issues/419

**Comment by dystewart** [comment]
With the test cluster now at v4.15 we can look at upgrading the rhoai deployment in that cluster. As discussed in [https://github.com/nerc-project/operations/issues/547](https://github.com/nerc-project/operations/issues/547) let's move to a stable version of rhoai v2 by switching the operator subscription to that channel.

**Comment by dystewart** [comment]
Closing this issue and tracking future work in: [https://github.com/nerc-project/operations/issues/572](https://github.com/nerc-project/operations/issues/572)
